encoding.bs

```diff
@@ -1610,19 +1587,18 @@ constructor steps are:
  <a>this</a>'s <a for=TextDecoderCommon>encoding</a>, and set <a>this</a>'s
  <a for=TextDecoderCommon>stream</a> to a new <a for=/>stream</a>.
 
- <li><p>Let <var>startAlgorithm</var> be an algorithm that takes no arguments and returns nothing.
-
  <li><p>Let <var>transformAlgorithm</var> be an algorithm which takes a <var>chunk</var> argument
  and runs the <a>decode and enqueue a chunk</a> algorithm with <a>this</a> and <var>chunk</var>.
 
  <li><p>Let <var>flushAlgorithm</var> be an algorithm which takes no arguments and runs the
  <a>flush and enqueue</a> algorithm with <a>this</a>.
 
- <li><p>Let <var>transform</var> be the result of calling
- <a abstract-op>CreateTransformStream</a>(<var>startAlgorithm</var>, <var>transformAlgorithm</var>,
- <var>flushAlgorithm</var>).
-
- <li><p>Set <a>this</a>'s <a for=GenericTransformStream>transform</a> to <var>transform</var>.
+ <li><p>Set <a>this</a>'s <a for=GenericTransformStream>transform</a> to the result of
+ <a for=TransformStream>creating</a> a {{TransformStream}} with
```

Comment:
```suggestion
 <a for=TransformStream>creating</a> a {{TransformStream}} object with
```

Reply:
This mismatches the algorithm declaration, which does not include "object".